modules/mobile/lib/mobile/v0/exceptions/custom_errors.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'common/exceptions/base_error'
+
+module Mobile
+  module V0
+    module Exceptions
+      class CustomErrors < Common::Exceptions::BaseError
+        attr_reader :title, :body, :source, :telephone, :refreshable
+
+        def initialize(title:, body:, source:, telephone:, refreshable:)
+          @title = title
+          @body = body
+          @source = source
+          @telephone = telephone
+          @refreshable = refreshable
+          super
+        end
+
+        def errors
+          [
+            {
+              title: @title,
+              body: @body,
+              status: 418,
```

Comment:
Status should be populated from the locales. I think something like `i18n_data[:status]` will work.

Reply:
the status won't ever change. I've moved it to a constant but don't want to give a user adding this error the opportunity to fat thumb the status code

Reply:
I think we may be talking about different things. I'm saying that by using the i18n method, you'll be using the error data you added to `modules/mobile/config/locales/exceptions.en.yml`. Which effectively acts as a constant. This is part of the established pattern that vets-api error classes use.

Reply:
oh got it. sure, added `i18n_data[:status]`